src/xls.rs

```diff
@@ -455,10 +461,60 @@ impl<RS: Read + Seek> Xls<RS> {
     }
 }
 
+/// https://learn.microsoft.com/en-us/openspecs/office_file_formats/ms-xls/4d6a3d1e-d7c5-405f-bbae-d01e9cb79366
+struct Bof {
+    /// Binary Interchange File Format
+    biff: u8,
+}
+
+/// BOF [MS-XLS] 2.4.21
+fn parse_bof(r: &mut Record<'_>) -> Result<Bof, XlsError> {
+    let mut dt = 0;
+    let biff_version = read_u16(&r.data[..2]);
+
+    if r.data.len() >= 2 {
+        dt = read_u16(&r.data[2..]);
+    };
+
+    let mut biff = match biff_version {
+        // BIFF8
+        0x0600 => 8,
+        // BIFF5
+        0x0500 => 5,
+        // BIFF4
+        0x0400 => 4,
+        // BIFF3
+        0x0300 => 3,
+        // BIFF2
+        0x0200 | 0x0002 | 0x0007 => 2,
+        _ => {
+            if r.data[4..].len() > 6 {
+                return Err(XlsError::Len {
+                    typ: "BOF data length",
+                    expected: 6,
+                    found: r.data.len(),
+                });
+            } else {
+                8
+            }
+        }
+    };
+
+    if biff_version == 0 && dt == 0x1000 {
```

Comment:
1. How come a biff_version = 0 is valid? Even looking at spces from 2012 (the oldest one available on Microsoft website) I don't see it mentioned.
2. I think I'd prefer having these checks in the `_` branch only
3. I am not sure where this logic is coming from? I didn't find it in sheet.js?

Reply:
1. It doesn't appear that the BIFF0 is a valid version to me either. However it looks like this logic is related to how the beginning of BOF record is interpreted.
2. Agree! 
3. [This](https://github.com/SheetJS/sheetjs/blob/5b4806bff01a9f4289b4c04e8a60a6c4c52fe93a/xlsx.js#L19383) is where I found it. Not sure all of that logic is in agreement with the rest of the code I've seen in the same file though. I guess some of it is not making sense to me now. Let me clean it up a little.

Reply:
Thanks, 
As we do not have a proper test file for this and I don't understand why it is here, could we perhaps comment this part of the code, so if someone has an issue we can uncomment and put a proper test, in the meantime staying "true" to the spec seems like a better choice.